testing/libzookeeper/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Maciej Klak <klak.maciej@gmail.com>
+# Maintainer: Maciej Klak <klak.maciej@gmail.com>
+pkgname=libzookeeper
+pkgver=3.4.11
+pkgrel=0
+pkgdesc="The Apache ZooKeeper system for distributed coordination is a high-performance service for building distributed applications."
+url="https://zookeeper.apache.org"
+arch="all"
+license="Apache License"
+depends=""
+makedepends=""
+install=""
+subpackages="$pkgname-dev"
+source="ftp://ftp.task.gda.pl/pub/www/apache/dist/zookeeper/zookeeper-$pkgver/zookeeper-$pkgver.tar.gz"
```

Comment:
Wouldn't it be better to use an [HTTP](https://www.apache.org/dyn/closer.cgi#http) mirror here? For example travis does not allow downloads via FTP.

Reply:
Definitely, FTP should not be used in abuilds.